ex5/ex5.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Write a function that given a string, returns the first character that
+ * doesn't repeat in the whole string. Otherwise (all characters repeat),
+ * should return null.
+ *
+ * For example:
+ * firstUniqueChar("abcdedcba") === "e"
+ * firstUniqueChar("abcdeedcba") === null
+ * firstUniqueChar("abababcab") === "c"
+ * firstUniqueChar("aaaaaaaaa") === null
+ * firstUniqueChar("a") === "a"
+ */
+
+function firstUniqueChar(str) {
+
+	var doNotLookForCharList = '', searchChar = '', searchIndex = 0; 
+
+	for (searchIndex in str) {
+
+		searchIndex = parseInt(searchIndex);
+
+		searchChar = str.charAt(searchIndex);
+
+		//  Let's see if we did look for searchChar
+		if (doNotLookForCharList.includes(searchChar) != true) {
+
+			//  SearchChar is not in doNotLookForCharList so will search for it
+			if (str.indexOf(searchChar, searchIndex + 1) < 0) {
```

Comment:
It looks like you are searching ahead (with the indexOf) and before (using doNotLookForCharList). I don't think you need both methods.